tools/ci_build/build.py

```diff
@@ -1018,6 +1021,12 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
     else:
         add_cmake_define_without_override(cmake_extra_defines, "onnxruntime_PYBIND_EXPORT_OPSCHEMA", "OFF")
 
+    print('build.py: 3')
+    print(cmake_extra_defines)
```

Comment:
forget to remove them? :)

Reply:
Removed. Added for investigating @snnn's [idea ](https://github.com/microsoft/onnxruntime/pull/8652#issuecomment-899882057).